pkg/cloud/aws/scanner/scanner.go

```diff
@@ -60,13 +61,25 @@ func (s *AWSScanner) Scan(ctx context.Context, option flag.Options) (scan.Result
 		)
 	}
 
-	if len(option.RegoOptions.PolicyPaths) > 0 {
-		scannerOpts = append(
-			scannerOpts,
-			options.ScannerWithPolicyDirs(option.RegoOptions.PolicyPaths...),
-		)
+	var policyPaths []string
+	var downloadedPolicyPaths []string
+	if len(option.RegoOptions.PolicyPaths) == 0 {
```

Comment:
Shouldn't it be aligned with config scanning?

Reply:
good catch - fixed that here https://github.com/aquasecurity/trivy/pull/3015/commits/6565583a44934bf10b1e799ae289d8e909eac541